lib/compat/wordpress-6.0/block-template-utils.php

```diff
@@ -8,17 +8,17 @@
  */
 
 /**
- * Filters theme directories that should be ignored during export.
+ * Filters theme files and folders that should be ignored during export.
  *
  * @since 6.0.0
  *
  * @param string $path The path of the file in the theme.
- * @return Bool Whether this file is in an ignored directory.
+ * @return Bool Whether this file is ignored.
```

Comment:
I think this should be made more clear to align with the previous filter comment. Not a blocker though.

```suggestion
 * @return Bool Whether this file or directory is ignored.
```